File: RoadMap_WA/Data/Models/RoadMapNode.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoadMap_WA.Data.Models
{
    public class RoadMapNode
    {
        // Всякая дичь для базы данных
        /// <summary>
        /// Код Node в данной карте
        /// </summary>
        public int _Id { get; set; }
        /// <summary>
        /// ID Родительского Node
        /// </summary>
        public int _ParentID { get; set; }
        /// <summary>
        /// Тип Node-a (начало, посредник, конец)
        /// </summary>
        public string _Type { get; set; }

        // Реализация класса

        /// <summary>
        /// Название Node-a
        /// </summary>
        public string Title { get; set; }
        /// <summary>
        /// Длинное описание Node-a
        /// </summary>
        public string LongDesc { get; set; }

        public RoadMapNode(int id, int parentID, string type, string title, string longDesc)
        {
            _Id = id;
            _ParentID = parentID;
            _Type = type;
            Title = title;
            LongDesc = longDesc;
        }
    }
}

File: models/RoadMapNode.php
<?php

class RoadMapNode{
    public $ID; // ID пункта
    public $ParentID; // ID родительского пункта
    public $RoadMapID; // ID основной карты
    public $Type; // Тип Node-a (b = начальный, m = средний, e = конечный)
    public $Title; // Название пункта
    public $LongDesc; // Описание пункта

    public function __construct($id, $ParentID, $RoadMapID, $Type, $Title, $LongDesc){
        $this->ID = $id;
        $this->ParentID = $ParentID;
        $this->RoadMapID = $RoadMapID;
        $this->Type = $Type;
        $this->Title = $Title;
        $this->LongDesc = $LongDesc;
    }
}
